test/integration/model/create.test.js

```diff
@@ -1351,6 +1351,29 @@ describe(Support.getTestDialectTeaser('Model'), function() {
       });
     });
 
+    it('should not insert NULL for unused fields', function () {
+      var Beer = this.sequelize.define('Beer', {
+          style: Sequelize.STRING,
+          size: Sequelize.INTEGER,
+      });
+
+      return Beer.sync({force: true}).then(function () {
+        return Beer.bulkCreate([{
+            style: 'ipa'
+        }], {
+          logging: function(sql) {
+            if (dialect === 'postgres') {
```

Comment:
Really don't want to add more sql testing in the integration tests, we're trying to remove it as much as possible.